api/filter_dimensions.go

```diff
@@ -0,0 +1,372 @@
+package api
```

Comment:
I am not sure how this file has been added to my PR. I have merged in develop twice. It may be from that, or it could be deleted work that now is being folded back in. Please advise.